src/infrastructure/flows/archive_ratings.rs

```diff
@@ -33,9 +31,9 @@ pub fn post_archive_ratings(
     ids: &[&str],
 ) -> Result<()> {
     let connection = connections.shared()?;
-    let place_ids = connection.load_place_ids_of_ratings(ids)?;
+    let place_ids = connection.inner().load_place_ids_of_ratings(ids)?;
```

Comment:
This looks like an `impl Deref` is missing for `connection` to avoid the `.inner()` boilerplate.

Reply:
actually `.inner()` is not necessary anymore, because I had to write wrappers anyway.
`impl Deref` was not possible because of lifetime issues with the new type wrapper of `SqliteConnection`.